Fix Sphinx documentation build warnings

yenaing-oo opened the pull request:
Closes #461 

- Change doc settings for modules so that members without an explicit docstring will not be included within the docs. This means that any class attributes that you want to document should be included as part of the class's docstring.
- Removed "_static" from `html_static_path` in `docs.conf.py`, as this directory does not exist and the docs currently do not use custom static assets.

danloveg reviewed (changes requested):
I'm still getting a few errors and warnings when building the documentation locally with sphinx-build 7.4.7. You may need to remove the `_build` directory to see these issues because Sphinx seems to hide warnings and errors if they've already been encountered:

```
reading sources... [100%] settings/index
C:\Users\lovegrod\Documents\NCTR_Code\public\Secure-Record-Transfer\docs\running/index.rst:2: WARNING: Duplicate explicit target name: "podman".
C:\Users\lovegrod\Documents\NCTR_Code\public\Secure-Record-Transfer\docs\settings/index.rst:550: ERROR: Malformed table.
Text in column margin in table line 4.

======================================================  ======================================================  ======
Default in Dev                                          Default in Prod                                         Type
======================================================  ======================================================  ======
/opt/secure-record-transfer/app/media/uploaded_files/  /opt/secure-record-transfer/app/media/uploaded_files/  string
======================================================  ======================================================  ======
C:\Users\lovegrod\Documents\NCTR_Code\public\Secure-Record-Transfer\docs\settings/index.rst:545: ERROR: Error parsing content block for the "table" directive: exactly one table expected.

.. table::

    ======================================================  ======================================================  ======
    Default in Dev                                          Default in Prod                                         Type
    ======================================================  ======================================================  ======
    /opt/secure-record-transfer/app/media/uploaded_files/  /opt/secure-record-transfer/app/media/uploaded_files/  string
    ======================================================  ======================================================  ======
```

```
checking consistency... C:\Users\lovegrod\Documents\NCTR_Code\public\Secure-Record-Transfer\docs\admin/index.rst: WARNING: document isn't included in any toctree
C:\Users\lovegrod\Documents\NCTR_Code\public\Secure-Record-Transfer\docs\howtouse/aboutpage.rst: WARNING: document isn't included in any toctree
C:\Users\lovegrod\Documents\NCTR_Code\public\Secure-Record-Transfer\docs\howtouse/userprofile.rst: WARNING: document isn't included in any toctree
done
```

```
writing output... [100%] settings/index
C:\Users\lovegrod\Documents\NCTR_Code\public\Secure-Record-Transfer\docs\howtouse\transferform.rst:116: WARNING: undefined label: 'accepted file types'
```

(The command I ran was `sphinx-build docs docs/_build`)

yenaing-oo commented:
> You may need to remove the `_build` directory to see these issues because Sphinx seems to hide warnings and errors if they've already been encountered

That's good to know. I've fixed all errors/warnings now.

Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/463/commits/928cc308fb423c349c957d3d50ea9c2be6951681, https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/463/commits/bd17bfbd5b354081bbda72c86e6b5cacc2ed5a1a, https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/463/commits/2d9fea0e3189619291180a6b095ccbbb432447f8, https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/463/commits/51b4edf5b987f828c7509a5c84b39a177532aac2 and https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/463/commits/1bf934c11e2bd7c3a7715391ae10eb9258213a84

danloveg commented on docs/index.rst:
```diff
@@ -16,6 +16,7 @@ with `AtoM <https://accesstomemory.org/en/>`_.
    howtouse/index
    customization/index
    settings/index
+   admin/index
```
Can you remove this line and remove that admin folder entirely? We'll add something similar back in #471 (an issue I just opened), but right now this page is pretty useless

yenaing-oo replied:
For sure, I've done it here: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/463/commits/038546b42e12c64969234844ebc25de82b874f61

danloveg reviewed (changes requested):
Everything besides the one issue I flagged looks good
